apps/total-typescript/src/utils/get-og-image.ts

```diff
@@ -17,9 +17,8 @@ export const getOgImage = (options: OgImageUrlOptions) => {
     title: title,
   })
   const url =
-    (process.env.NODE_ENV === 'development' ? '' : CLOUDINARY_FETCH_BASE_URL) +
-    process.env.NEXT_PUBLIC_OG_IMAGE_URI +
-    `?${query.toString()}`
+    // (process.env.NODE_ENV === 'development' ? '' : CLOUDINARY_FETCH_BASE_URL) +
+    process.env.NEXT_PUBLIC_OG_IMAGE_URI + `?${query.toString()}`
```

Comment:
@vojtaholik in Vercel we have this value for `NEXT_PUBLIC_OG_IMAGE_URI`.
![CleanShot 2023-12-21 at 18 32 13@2x](https://github.com/skillrecordings/products/assets/1519448/d7fbac35-c1a4-46f8-b94a-23e12c9ec8c4)
Not sure if it works as expected. It seems we have to change it for the `https://totaltypescript.com/api/og`.

Reply:
@DrShpongle yes that's an old unused one, I reckon `next_public_url + path` would be better here.